easybuild/framework/easyconfig/tools.py

```diff
@@ -322,6 +322,11 @@ def alt_easyconfig_paths(tmpdir, tweaked_ecs=False, from_prs=None):
     # paths where files touched in PRs will be downloaded to,
     # which are picked up via 'pr_paths' build option in fetch_files_from_pr
     pr_paths = None
+    if from_prs and review_pr:
+        from_prs.append(review_pr) if review_pr not in from_prs else from_prs
```

Comment:
@jfgrimm Please update to:

```python
if from_prs and review_pr and review_pr not in from_prs:
    from_prs = from_prs + [review_pr]
```

Two reasons:

* We shouldn't append to the `from_prs` list that is passed to `alt_easyconfig_paths`, since that's a side effect (that change will stick even outside of the scope of the `alt_easyconfig_paths` call);
* The `if ... else` construct here doesn't make sense, in particular the `else from_prs` part doesn't (since we're not assigning the result of that expression)